public/scripts/controllers/camp_manage.js

```diff
@@ -45,15 +45,23 @@ app.controller("manageCampsController", function ($scope, $http, $filter) {
     }
 
     // update the camp pre sale quota
-    $scope.updatePreSaleQuota = (camp_id,quota) => {
-        confirm('Confirm new quota to: ' + quota);
-        $.post('/camps/' + camp_id + '/updatePreSaleQuota', { quota: quota })
-           .success(function(response) {
-            window.location.reload();
-        })
-        .error(function() {
-            alert("Quota must be in a positive number format");
-        }); 
+    $scope.updatePreSaleQuota = (camp_id, quota) => {
+        if (typeof currentEvent === 'function') {
+            currentEvent();
+        }
+
+        let current = new Date();
+        let start = new Date(currentEvent.addinfo_json.start_presale_tickets_allocation);
+        let end = new Date(currentEvent.addinfo_json.end_presale_tickets_allocation);
+        if (start < current && current < end) {
+            if (confirm('Confirm new quota to: ' + quota)) {
+                $.post('/camps/' + camp_id + '/updatePreSaleQuota', { "quota": quota })
+                    .success(() => { })
+                    .error(() => {
+                        alert("Quota must be in a positive number format");
+                    });
+            }
+        }
     }
```

Comment:
changing the camp presale allocation will open a confiramtion box only inside the valid allocation date period